lib/make-middleware.js

```diff
@@ -136,6 +136,7 @@ function makeMiddleware (setup) {
         })
 
         fileStream.on('limit', function () {
+          fileStream.emit('close')
```

Comment:
I think that theses events are only meant to be emitted internally by the streams. The correct way would probably be to `unpipe` the incoming request stream 🤔

Reply:
Is there any way for someone else to make this change, since @DiegoRBaquero seems busy?

Reply:
I need this

Reply:
@LinusU would you mind making the change? I couldn't figure it out. change on disk would be to listen to `unpipe` event instead of close.

Reply:
I'm not sure exactly what should be changed though, I think that this requires a lot of testing to get right 🤔